multimedia/whispercpp/Makefile

```
@@ -0,0 +1,48 @@
+#
+# Copyright (C) 2024 Dengfeng Liu <liudf0716@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v3.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=whispercpp
+PKG_VERSION:=1.5.4
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.gz
```

```suggestion
PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
```

`v` is usually omitted in our usual practice, but it's okay to keep it as-is.